shared/chat/conversation/attachment-video-fullscreen/index.desktop.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React from 'react'
+import {WebView} from '../../../common-adapters'
+import {globalColors} from '../../../styles'
+import type {Props} from './index.types'
+
+// Unused for now, we don't view videos in the app on desktop
```

Comment:
? is this true? why return webview?

Reply:
Desktop it just downloads it, doesn't do the view in here. Is there something smaller I can return from here? I just wanted to satisfy flow.